app/forms/forms/mark_complete_form.rb

```diff
@@ -21,4 +22,12 @@ def assign_form_values
     self.mark_complete = form.try(:question_section_completed)
     self
   end
+
+  def marked_complete?
+    mark_complete == "true"
+  end
+
+  def has_validation_errors
```

Comment:
should this maybe be called `has_routing_errors`? I'm only asking because it feels like we are introduction a whole new concept here like whats the difference between validation errors and routing errors? Dont have strong opinions on this only a suggestion.